source/reference/command/replSetGetStatus.txt

```diff
@@ -195,23 +195,13 @@ following fields:
 
    .. versionadded:: 3.4
 
-   The ``optimes`` field holds a document that contains information on
-   various optimes. Optimes are a reference to a position in the
-   replication "oplog", and are reported here to show replication
-   progress for various operations.
-   
-   If using :rsconf:`protocolVersion: 1 <protocolVersion>`, the optime
-   value is a document that contains:
-
-   - ``ts``, the :ref:`Timestamp <document-bson-type-timestamp>` of
-     the operation.
+   The ``optimes`` field holds a document that contains
+   :term:`optimes <optime>` showing replication progress for various
+   types of operations.
```

Comment:
So, it's the second point:
optimes showing replication progress ...  makes it sound like there are optimes that show one thing and other optimes showing some other optime.

Also, separately, both the original wording and new version for "various operations" and "various types of operations" -- I wonder if we should just get rid of it as  it sounds like there are different optimes for updates or something.